web-nfc.d.ts

```diff
@@ -0,0 +1,90 @@
+// Type definitions for Web NFC
+// Project: https://github.com/w3c/web-nfc
+// Definitions by: Takefumi Yoshii <https://github.com/takefumi-yoshii>
+// TypeScript Version: 3.9
+
+// This type definitions referenced to WebIDL.
+// https://w3c.github.io/web-nfc/#actual-idl-index
+
+interface Window {
+  NDEFMessage: NDEFMessage
+}
+declare class NDEFMessage {
+  constructor(messageInit: NDEFMessageInit)
+  readonly records: NDEFRecord[]
```

Comment:
[IMO] IDL says it's `FrozenArray<T>` rather than `sequence`. Should this be `ReadonlyArray<Readonly<NDEFRecord>>` then ?

Reply:
I don't know what kind of type is used if typings were auto generated from IDL, and it's okay to keep this if they would use `T[]`.

Reply:
I think `ReadonlyArray<NDEFRecord>` should be fine.